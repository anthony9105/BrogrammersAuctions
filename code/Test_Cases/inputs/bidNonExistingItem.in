login adminUser1
bid large_table user1234
