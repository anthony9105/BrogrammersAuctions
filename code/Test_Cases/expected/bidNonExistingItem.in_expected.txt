Login successful
Error. Item does not exist
